tests/integration/go_ethereum/conftest.py

```diff
@@ -97,12 +97,16 @@ def base_geth_command_arguments(geth_binary, datadir):
         "2",
         "--password",
         os.path.join(datadir, "keystore", "pw.txt"),
+        # in order to raise on underpriced transactions, ``txpool.nolocals`` is now
+        # necessary: https://github.com/ethereum/go-ethereum/pull/31202
+        "--txpool.nolocals",
```

Comment:
should we add this to `geth_kwargs` in `py-geth`?

Reply:
I had that thought too. I don't see why not if it's a quick add.